src/api/checkout/checkout.controller.ts

```diff
@@ -22,11 +24,26 @@ export const checkoutHandler = async (req: Request, res: Response) => {
       }
     })
 
+    let orderStatus = 'pending-for-payment'
+    if (payment?.status === "succeeded") {
+      orderStatus = 'success'
+    }
+
+    const order = await OrderService.createOrder(req.user, {
+      ...checkout,
+      userId: req.user?.id,
+      status: orderStatus,
+      paymentId: payment?.id,
+      paymentMethod: payment?.payment_method_types[0]
+    }, products)
+
     res.send({
-      message: 'Hello from checkout',
-      payment
+      message: 'The order was created successfully',
+      payment,
+      order,
     });
   } catch (error: any) {
+    console.log(error)
```

Comment:
remove this console.log

Reply:
Hola hector a mi me interesa tener registro del error  en el servidor y no tenemos sistema de logger de errores entonces por ahora es la mejor manera de tener ese control de los errores